### ISSUE TITLE
Collapse Accessibility

### ISSUE THREAD

**Issue by TheSharpieOne** [reactstrap/reactstrap#1627]
https://getbootstrap.com/docs/4.0/components/collapse
https://getbootstrap.com/docs/4.0/components/collapse/#accessibility

missing `aria-expanded`

we cannot really control the `aria-controls` for the triggering element, it would need to be up to the developer to add an `id` to the collapse container and specific that `id` as the `aria-controls` prop for the triggering element
missing id for collapse container (unless someone has a better idea (without needing some wrapping component))

#655
